packages/tofu/test/inspectEsmImports.spec.js

```diff
@@ -29,22 +29,26 @@ testInspect(
   import defaultExport3, * as name2 from "package09";
   import "package10";
   var promise = import("package11");
+  export * as name3 from "package12";
+  export { name4 } from "package13";
+  export { default as name5 } from "package14";
+  export const foo = "bar";
```

Comment:
a) Treating re-exports as imports may be semantically incorrect and b) there may be some other reason to not do this.  

Respective to a) I don't know why this would be true, and b) I don't know what that would be.